packages/server-wallet/src/config/defaults.ts

```diff
@@ -46,24 +41,29 @@ export const defaultMetricsConfiguration = {timingMetrics: false};
  */
 export const defaultConfig: OptionalServerWalletConfig = {
   databaseConfiguration: defaultDatabaseConfiguration,
-  networkConfiguration: defaultNetworkConfiguration,
   loggingConfiguration: defaultLoggingConfiguration,
   metricsConfiguration: defaultMetricsConfiguration,
-  // TODO: List addresses this corresponds to
-  ethereumPrivateKey: '0x7ab741b57e8d94dd7e1a29055646bafde7010f38a900f55bbd7647880faa6ee8',
   skipEvmValidation: false,
-  workerThreadAmount: 10,
```

Comment:
IMO this could reasonably be a default, but the value should probably be `0`?

Reply:
Done